File: solutions/uri/1005/1005.cs
using System;
using System.Collections.Generic;

class Solution {
    static void Main() {
        double a, b;
        string val;

        val = Console.ReadLine();
        a = Convert.ToDouble(val);
        val = Console.ReadLine();
        b = Convert.ToDouble(val);

        Console.Write(
            "MEDIA = {0:F5}{1}",
            (a * 3.5 + b * 7.5) / 11.0,
            Environment.NewLine
        );
    }
}

File: solutions/uri/1005/1005.scala
object Main {
    def main(args: Array[String]) {
        val a = io.StdIn.readLine().toDouble
        val b = io.StdIn.readLine().toDouble

        println("MEDIA = %.5f".format((a * 3.5 + b * 7.5) / 11.0))
    }
}
